1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -14,6 +14,15 @@ let salaries = {
 
 // WRITE BELOW THIS LINE
 
+for( let salary in salaries ) {
+  let namedPerson = salaries[salary]
+  if(namedPerson >= 50000) {
+  console.log(salary + ':' + ' $' + salaries[salary])
```

Comment:
Hi Ika, well done for all your effort,you are doing really great. I think you should consider using 'string interpolation' for your console. I makes the code more readable.
`console.log(`${salary}: $${salaries[salary]}`)`